_posts/2020-05-13-the-hard-path-to-communicating-while-remote.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title: The Hard Path to Communicating While Remote 
+summary:
+  The pandemic can lead us to better modes of communication, if we embrace the challenge
+author: Eric Tillberg
+author_url: http://teecom.com/people/eric-tillberg/
+---
+
+When presented with a new constraint on how we perform habitual tasks, we often
+have a choice: force our way of working into the new reality or learn from the
+new reality and adapt. The distinction between these two options can sometimes
+get blurry. However, our current situation is unusually clear. Our new
+constraint is that all thought workers are now working remotely. How we cope
+with this constraint can help us not only during this pandemic but also
+afterwards. Secondly, it will have an impact on how our physical work spaces are
+organized and subdivided.
+
+Video conferencing has displaced in-person gatherings of all kinds, business and
+social, formal and informal. Unfortunately, video conferencing is nothing more
+than our best approximation of the in-person meeting. If we were learning from
+the new reality and adapting, our solution would look less like an approximation
+of the status quo and more like something truly novel.
+
+When our working routines depend on in-person meetings, then video conferencing
+can be a life-saver. On the contrary, I doubt many people would claim to enjoy
+in-person meetings purely for their own sake. Indeed, it has become a painful
+cliché in office settings that meetings are a waste of time. This leads me to
+the conclusion that we’re forcing our way of working into the new reality.
+Zoom’s explosion in daily users, from 10 million to 200 million, tells us a
+convincing story that we don’t know what to do with this new constraint.
+
+Instead of settling for a bandwidth-choked imperceptibly-laggy audio-visual
+approximation of an in-person meeting, we should think about the ways in which
+remote work pushes us into different habits. We can learn from the constraint
+and actually stand to benefit from it. Co-located work leads us to favor spoken
+conversation as our decision-making technique because it is just so easy to tap
+someone on the shoulder. Remote work ought to lead us to written discussion as
+our favored decision-making technique because our environment is more conducive
+to deep, uninterrupted thought. The imposition of asking someone spontaneously
```

Comment:
What about extroverts who thrive on spontaneous interaction?

Reply:
Point well taken, but I think I will leave this as-is. I think it's ultimately a big tug-of-war between introverted and extroverted defaults and my own perception is that the open office gives a lot to the primacy of extroverts. Sure, flipping to the opposite extreme threatens to diminish extroverts but I hope I'm advocating more flexibility in considering what we use as a "default" rather than starkly taking sides. But it's a really good point and one that I probably don't adequately address here...